ZackHolmberg/COMP4350-Project
# 144: Replication and backup

### Summary of changes
- Added a new backup container
- replicate function that replicates request which change the state of main container to the backup container
- Master container pulls chain and wallets on startup from the backup container
- Unit tests 

### How to test
- run the containers, make a few requests
- stop the main container
- make a few requests on the same route `localhost\blockchain`
- start the main container and check the persistence


Closes #144
